scripts/cond_patch.py

```diff
@@ -38,8 +38,54 @@ def apply_patch(ppath, repo_folder, level):
       '-r', '-', '-s', '--no-backup-if-mismatch'
   ])
 
+def fetch_branch(remote, branch, repo_folder):
+  return subprocess.call([
+    'git', 'fetch', remote, branch
+  ], cwd=repo_folder)
+
+def checkout_branch(branch, repo_folder):
+  return subprocess.call([
+    'git', 'checkout', branch
+  ], cwd=repo_folder)
+
+def apply_companion(repo_folder, patch_folder, depth, remote):
```

Comment:
But all this can be achieved with a simple `git checkout foo` on the pytorch repo (which is what I do), right?
I honestly do not see value in doubling the size of this script just to avoid a single command.

Reply:
Which command to avoid?